auto_round/utils.py

```diff
@@ -2070,3 +2073,146 @@ def out_of_vram(error_msg):
     if "HIP out of memory. Tried to allocate" in error_msg:
         return True
     return False
+
+
+def is_moe(module: torch.nn.Module) -> bool:
+    """Returns whether the module is an MOE layer."""
+    return any(
+        key in type(module).__name__.lower()
+        for key in [
+            "MixtralSparseMoeBlock".lower(),
+            "ArcticMoE".lower(),
+            "DbrxFFN".lower(),
```

Comment:
how about checking the config, the key should contains some "experts"

Reply:
Will try to optimize this func at next PR